pkg/render/render.go

```diff
@@ -71,7 +71,7 @@ func RenderTemplate(path string, d *RenderData) ([]*unstructured.Unstructured, e
 	tmpl.Funcs(template.FuncMap{"getOr": getOr, "isSet": isSet})
 	tmpl.Funcs(sprig.TxtFuncMap())
 
-	source, err := ioutil.ReadFile(path)
+	source, err := ioutil.ReadFile(filepath.Clean(path))
```

Comment:
This also introduces a bug. If the path contains parent-directory marks, then we absolutely should follow them.

If we wanted to reject them (which we don't), then as a general rule, silently removing them is the wrong way to go about it. Instead, you should report a failure to the user and halt the program.